docs/conf/au/2017/news/event-recap-feedback

```diff
@@ -0,0 +1,143 @@
+:template: 2017/au-content.html
+
+.. post:: Dec 1, 2017
+   :tags: 2017, australia
+
+Write the Docs Day Australia 2017 - Recap
+=================================================
+
+A big thank you to all attending the 2017 and our first Australian Write the
```

Comment:
A bit awkward. Perhaps:

    A big thank you to all attending the first Write the Docs Day in Melbourne.